src/main/java/org/myrobotlab/service/Runtime.java

```diff
@@ -3074,7 +3274,11 @@ public Object sendBlockingRemote(Message msg, Integer timeout) throws IOExceptio
       // --> remote
       // FIXME implement !
       Endpoint endpoint = (Endpoint) conn.get("c-endpoint");
-      endpoint.socket.fire(CodecUtils.toJson(msg));
+      String json = CodecUtils.toJson(msg);
+      if (json.length() > 65536) {

```

Comment:
why do we have a 64k limit here?  is that some protocol limitation?

Reply:
oop ... nice spot check ..
Not sure if its a websocket RFC spec or a safety measure employed by the Atmosphere implementation. 

By default using atmosphere, if the message is over 65K it gets dumped.
I think overall its a good idea to try to keep the message payloads small, so its a bit of a reminder.  With configuration I made it have more capacity, but I still want to slice down a couple bloat messages.

I'll rip out this debugging stuff

Reply:
I was writing out the large messages and inspecting them (too that out) - I left the warnings in - there could be more refactoring done with the bloated messages